### DISCUSSION TITLE
Can h2o.llmstudio run on alternative hardware?

### DISCUSSION THREAD

**Discussion by flippercy** [h2oai/h2o-llmstudio#189] [General]
Hi:

Can I use other type of GPUs to run h2o.llmstudio or do I must use what specified in the instruction?

"H2O LLM Studio requires a machine with Ubuntu 16.04+ and at least one recent Nvidia GPU with Nvidia drivers version >= 470.57.02"

Right now I have no recent Nvidia GPU available......

Thank you.

**Comment by tmm1** [comment]
I think it requires nvidia GPU. That is true of most ML projects and most code which uses pytorch.

You could rent a GPU on vast.ai and run the llmstudio docker. I was able to do that and it works.

**Comment by pascal-pfeiffer** [comment]
Correct, only NVIDIA GPUs are supported currently.
